docs/op-guide/horizontal-scale.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Scale a TiKV Cluster
+summary: Learn how to add or delete a PD/TiKV node.
+category: operations
+---
+
+# Scale a TiKV Cluster
+
+You can increase or decrease the capacity of a TiKV cluster without affecting online services. This document describes how to add or delete a PD or TiKV node.
```

Comment:
How about `scale in` and `scale out`?

Reply:
@rleungx You mean this sentence? "This document describes how to add or delete a PD or TiKV node."